models/user/user.go

```diff
@@ -745,6 +750,22 @@ func createUser(ctx context.Context, u *User, createdByAdmin bool, overwriteDefa
 		return err
 	}
 
+	if setting.RecordInitialIP {
+		// insert initial IP and UserAgent
+		if err = SetUserSetting(ctx, u.ID, "initial_ip", u.InitialIP); err != nil {
+			return err
+		}
+
+		// trim user agent string to a reasonable length, if necessary
+		userAgent := strings.TrimSpace(u.InitialUserAgent)
+		if len(userAgent) > 255 {
+			userAgent = userAgent[:255]
+		}
+		if err = SetUserSetting(ctx, u.ID, "initial_user_agent", userAgent); err != nil {
```

Comment:
There are already consts, but not magic strings.

![image](https://github.com/user-attachments/assets/08fa0ac5-377e-4bf6-a7e2-fc17201610ea)